Mage.Client/src/main/java/org/mage/plugins/card/utils/impl/ImageManagerImpl.java

```diff
@@ -345,6 +345,15 @@ public Image getSkipEndStepBeforeYourTurnButtonImage() {
         return imageSkipUntilEndStepBeforeYourTurnButton;
     }
 
+    @Override
+    public Image getSkipEachEndStepButtonImage() {
+        if (imageSkipUntilEachEndStepButton == null) {
+            imageSkipUntilEachEndStepButton = getBufferedImageFromResource(
+                    PreferencesDialog.getCurrentTheme().getButtonPath("skip_to_each_end.png"));
```

Comment:
Why does the text in 3/4 images "to all end" and 1/4 says "to each end"?

Reply:
There was no sample of all possible letters given by whomever made the buttons, and it was easier to make them say all.

> > Hi. This allows the user to choose to skip to the end step of each step (including their own). F5 doesn't actually just stop at the end step and forces you to respond to triggers. In multiplayer games and for decks that play, for example, seedborn muse/unwinding clock, you want to be able to ignore the turn until you are able to do something at the end of each turn. This will actually speed up games and is an enhancement on the F11 function I implemented in like 2016 or so.
> 
> I see, thanks for the clarification. That makes sense. Yes I agree it would be useful to have a version of F5 that doesn't stop for triggers on the stack.
> 
> I seen this as a variation of F11 AND F5. F11 since it stops on all ENDs and F5 since it skips choices (other than blockers).
> 
> Is it possible to change one of them to work something like: F5 works as normal, SHIFT+F5 skips stack items? (or same idea for F11)
F11 works as it should in most circumstances (aka, I would like to do stuff just prior to my turn).  F5 does.. whatever F5 does..  I feel this is a unique enough function that it deserves its own hot key and there's no other hotkey that has that type of modifier added to it.

Reply:
Can potentially change one or the other three to read something all (or each).

Reply:
I think there needs to be a better differentiation between it at F5. Because F5 can also be changed to do both. See my comment above about potentially adding in a combo shortcut: https://github.com/magefree/mage/pull/9504#issuecomment-1242835467